client/src/controllers/DrilldownController.ts

```diff
@@ -1,10 +1,62 @@
 import { Controller } from '@hotwired/stimulus';
-import { ActionController } from './ActionController';
-import { DropdownController } from './DropdownController';
+import type { ActionController } from './ActionController';
+import type { DropdownController } from './DropdownController';
 
 /**
  * Drilldown menu interaction combined with URL-driven
  * state management for listing filters.
+ * 
+ * @example
+ * ```html
+ *    <section>
+            <div id="filters-drilldown" className="w-drilldown" data-controller="w-drilldown" data-action="w-swap:success@document->w-drilldown#updateCount w-dropdown:hide->w-drilldown#delayedClose w-dropdown:clickaway->w-drilldown#preventOutletClickaway"
```

Comment:
Please include the `*` and try to nest these with two spaces only, it's much easier to read.
Please also remove all but the bare minimum DOM that's needed to show an example. Try to find a minimal reproducible usage, test it yourself (in storybook or in the admin) and then use that as an example.

We should not need any classes (or just those that help differentiate elements), and a reminder that it's `class` not `className` in HTML.

```suggestion
 *   <div id="filters-drilldown" className="w-drilldown" data-controller="w-drilldown" data-action="w-swap:success@document->w-drilldown#updateCount w-dropdown:hide->w-drilldown#delayedClose w-dropdown:clickaway->w-drilldown#preventOutletClickaway"
```